### PR TITLE
Si 19047

### PR DESCRIPTION
**Pull request by phayes-r7**
## Description
A detailed description of your changes.

## Motivation and Context
Explanation of why these changes are being proposed, including any links to other relevant issues or pull requests.

## How Has This Been Tested?
A clear and concise description of your changes were tested.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly (or changes are not required).
- [ ] I have added tests to cover my changes (or new tests are not required).
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java [hunk 1]
```diff
@@ -51,7 +52,11 @@ public Set<Package> parse(File input, OperatingSystem operatingSystem) throws IO
           String name = getValueForAttribute(element, "id");
           String version = getValueForAttribute(element, "version");
           String description = getValueForAttribute(element, "description");
-          packages.add(new Package(source, PackageType.DOTNET, operatingSystem, name, version, description, null, null, null, null));
+          try {
+            packages.add(new Package(source, PackageType.DOTNET, operatingSystem, name, version, description, null, null, null, null));
+          } catch (PackageValidationException pve) {
+            LOGGER.info(pve.getMessage());
```
Comment (bholden-r7) [inline src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java:58]: Do you really want this at INFO level?

File: src/main/java/com/rapid7/container/analyzer/docker/packages/M2RepositoryParser.java [hunk 2]
```diff
@@ -38,17 +42,13 @@ public Set<Package> parse(File input, OperatingSystem operatingSystem) throws IO
...
-        null,
-        0L,
-        null,
-        null,
-        null));
+    try {
+      return singleton(new Package(fileName, PackageType.JAVA, null, format(PACKAGE_NAME_FORMAT, groupId, artifactId), versionNumber, null, 0L, null,
+          null, null));
+    } catch (PackageValidationException pve) {
+      LOGGER.info(pve.getMessage());
```
Comment (bholden-r7) [inline src/main/java/com/rapid7/container/analyzer/docker/packages/M2RepositoryParser.java:49]: Same question

**Review by bholden-r7 — APPROVED** [review]

File: src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java (hunk 1 above)
Comment (phayes-r7) [reply src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java:58]: It happens a lot is the thing. I thought about putting it at warn, but chances are the people using the app aren't going to change anything based off that warning. I'm kinda on the fence about it. What do you think?

Comment (bholden-r7) [reply src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java:58]: Like how often? I agree with your second point but it is a bit weird to cover up bad things with info logging

**Pull request merged**